package.json

```diff
@@ -9,39 +9,70 @@
   "license": "MIT",
   "homepage": "https://github.com/jsonapi-suite/jsorm",
   "scripts": {
-    "clean": "shx rm -rf _bundles lib lib-esm",
-    "build": "shx cp es-shims/* src/util/ && npm run clean && tsc && tsc -m es6 --outDir lib-esm && webpack && shx cp node-shims/* src/util/ ",
-    "test": "mocha -r test/tshook.js test/**/*.ts || true"
+    "f:clean": "shx rm -rf _bundles lib lib-esm",
+    "f:build": "shx cp es-shims/* src/util/ && npm run clean && tsc && tsc -m es6 --outDir lib-esm && webpack && shx cp node-shims/* src/util/ ",
+    "f:test": "mocha -r test/tshook.js test/**/*.ts || true",
+    "prebuild": "tsc -t es5 -m es6 --skipLibCheck -p ./tsconfig.json",
+    "build": "npm-run-all --parallel build:dev build:prod",
+    "build:dev": "NODE_ENV=development ./node_modules/.bin/rollup -c",
+    "build:prod": "NODE_ENV=production ./node_modules/.bin/rollup -c",
+    "build:watch": "NODE_ENV=watch yarn run tsc --watch -p .",
+    "coverage": "npm run pretest && node --harmony ./node_modules/istanbul/lib/cli.js cover --root ./build/src --report lcov --report text ./node_modules/mocha/bin/_mocha ./build/test/**/*.js",
+    "clean": "rimraf ./dist ./build ./coverage ./lib",
+    "test": "NODE_ENV=test mocha --opts test/mocha.opts",
+    "test:jsdom": "NODE_ENV=test mocha --require test/setup/setup-dom.js 'test/**/*.test.ts'",
```

Comment:
Do we have these tests? Maybe need to audit these and remove anything this project isn't actually using.

Reply:
Yeah all of this was pulled over from my rollup sample project.  Haven't cleaned up everything yet and this probably goes away.  I'm ignoring all build and dependency-related comments for the rest of this, since all of that will get a pre-release focus.